csi/util.go

```diff
@@ -460,3 +461,28 @@ func requiresSharedAccess(vol *longhornclient.Volume, cap *csi.VolumeCapability)
 func getStageBlockVolumePath(stagingTargetPath, volumeID string) string {
 	return filepath.Join(stagingTargetPath, volumeID)
 }
+
+// We use this utility function because csi-plugin has no datastore object to ask about kubernetes node,
+// and the go-common-lib/ns functions won't work because the /proc mount path is different.
+func getOSInfo() string {
+	utsname := &syscall.Utsname{}
+	if err := syscall.Uname(utsname); err != nil {
+		logrus.WithError(err).Warn("Failed making syscall Uname.")
+		return "<unknown OS>"
+	}
+
+	extract := func(field [65]int8) string {
```

Comment:
Why 65 here?

Reply:
From the golang syscall package, the Utsname struct is
```go
type Utsname struct {
	Sysname    [65][int8](https://pkg.go.dev/builtin#int8)
	Nodename   [65][int8](https://pkg.go.dev/builtin#int8)
	Release    [65][int8](https://pkg.go.dev/builtin#int8)
	Version    [65][int8](https://pkg.go.dev/builtin#int8)
	Machine    [65][int8](https://pkg.go.dev/builtin#int8)
	Domainname [65][int8](https://pkg.go.dev/builtin#int8)
```